packages/nxengine/fix-prefix.patch

```diff
@@ -0,0 +1,19 @@
+diff -uNr a/Makefile.linux b/Makefile.linux
+--- a/Makefile.linux
++++ b/Makefile.linux
+@@ -16,11 +16,11 @@
+ CXX           = g++
+ DEFINES       = # -D_L10N_CP1251
+ CFLAGS        = -pipe -O2 $(DEFINES)
+ CXXFLAGS      = -pipe -O2 $(DEFINES)
+-INCPATH       = -I/usr/include -I/usr/include/SDL -I/usr/local/include -I/usr/local/include/SDL -I.
++INCPATH       = -I$TERMUX_PREFIX/include -I$TERMUX_PREFIX/include/SDL -I$TERMUX_PREFIX/local/include -I$TERMUX_PREFIX/local/include/SDL -I.
+ LINK          = g++
+-LFLAGS        = -Wl,-O1 -Wl,-rpath,/usr/lib
++LFLAGS        = -Wl,-O1 -Wl,-rpath,$TERMUX_PREFIX/lib
```

Comment:
Why not to just use flags from LDFLAGS? Similar issue with INCPATH...

Reply:
P.S. Compilers and other related vars should be retrieved from environment variables too.